### PR TITLE
Minor rewrite of title and description text for CKEditor toolbar customization demo

### PR DESCRIPTION
**Pull request by TheRealEdDawson**
Info 233/edit text for ckeditor toolbar customization demo

### CODE REVIEW

File: www/authoring/custom-ckeditor-toolbar.php [hunk 1]
```diff
@@ -76,10 +76,12 @@
...
         <div class="overview">
-            <h2>Customizing the RTE toolbar</h2>
-            <p>Customize the RTE toolbar to rearrange or hide specific options.</p>
-            <p>This demo demonstrates how you can customize the RTE toolbar to hide specific options which should not be made available during authoring.</p>
-            <p>See an article on how to <a href="https://help.learnosity.com/hc/en-us/articles/360014562597-Customizing-the-RTE-toolbar-in-Question-Editor-API">customize the RTE toolbar in Question Editor API</a>.</p>
+            <h2>Customizing the Rich Text Editor toolbar</h2>
+            <p>You can customize the Rich Text Editor (RTE) toolbar to rearrange or hide specific options for authors. For example, showing fewer buttons for a cleaner interface.</p>
+            <p>By default, the toolbar shows 28 buttons, like this:  
+            <img src="../static/images/ckeditor-default-toolbar.png" height=60% width=60%></p>
+            <p>In this demo, we have customized the toolbar to only show seven buttons, like so: <img src="../static/images/ckeditor-customized-toolbar.png"  height=15% width=15%></p>
```
Comment (hiaik) [inline www/authoring/custom-ckeditor-toolbar.php:83]: Just out of the interest @TheRealEdDawson why did you use "seven buttons" here (seven) as a word and "28 buttons" above where 28 is a number?

File: www/authoring/custom-ckeditor-toolbar.php [hunk 2]
```diff
@@ -76,10 +76,12 @@
...
         </div>
         <div class="overview">
-            <h2>Customizing the RTE toolbar</h2>
-            <p>Customize the RTE toolbar to rearrange or hide specific options.</p>
-            <p>This demo demonstrates how you can customize the RTE toolbar to hide specific options which should not be made available during authoring.</p>
-            <p>See an article on how to <a href="https://help.learnosity.com/hc/en-us/articles/360014562597-Customizing-the-RTE-toolbar-in-Question-Editor-API">customize the RTE toolbar in Question Editor API</a>.</p>
+            <h2>Customizing the Rich Text Editor toolbar</h2>
+            <p>You can customize the Rich Text Editor (RTE) toolbar to rearrange or hide specific options for authors. For example, showing fewer buttons for a cleaner interface.</p>
+            <p>By default, the toolbar shows 28 buttons, like this:  
+            <img src="../static/images/ckeditor-default-toolbar.png" height=60% width=60%></p>
```
Comment (hiaik) [inline www/authoring/custom-ckeditor-toolbar.php:82]: @TheRealEdDawson  Do we want to re-capture that one without "text left-align" to be in focus?

**Review by hiaik — CHANGES_REQUESTED** [review]

File: www/authoring/custom-ckeditor-toolbar.php (hunk 1 above)
Comment (TheRealEdDawson) [reply www/authoring/custom-ckeditor-toolbar.php:83]: It's a convention in professional communications, based on the Chicago Manual of Style. You'll see the same thing in newspapers and good quality journalism.

File: www/authoring/custom-ckeditor-toolbar.php (hunk 2 above)
Comment (TheRealEdDawson) [reply www/authoring/custom-ckeditor-toolbar.php:82]: Oh, the image? Does it seem blurry for you?

Comment (hiaik) [reply www/authoring/custom-ckeditor-toolbar.php:82]: I meant focus on a single button... Discussed in slack and we will go with what we have for now.

Comment (TheRealEdDawson) [reply www/authoring/custom-ckeditor-toolbar.php:82]: The blue selection from the left-align button being clicked, I see. I think it's maybe not a showstopper at this stage, but let's ask the Minions for a design review here. They might agree with your point.

**Review by hiaik — APPROVED** [review]

**Comment by hiaik** [comment]
LGTM;

**Comment by TheRealEdDawson** [comment]
Andy in the Minions team also had some commentary, here: https://learnosity.slack.com/archives/C11MX6SMP/p1604456284042400

**Pull request merged**